Runtime/BacktraceClient.cs

```diff
@@ -289,6 +292,10 @@ public static BacktraceClient Initialize(BacktraceConfiguration configuration, D
             var backtrackGameObject = new GameObject(gameObjectName, typeof(BacktraceClient), typeof(BacktraceDatabase));
             BacktraceClient backtraceClient = backtrackGameObject.GetComponent<BacktraceClient>();
             backtraceClient.Configuration = configuration;
+            if (attachments != null && attachments.Count != 0)
+            {
+                backtraceClient.Attachments = attachments;
```

Comment:
Is it safer to add an empty "attachments" object if we don't have any attachments from user?

Reply:
We did that - we auto initialize attachments on object startup just after constructor

Reply:
Ah okay I see, thank you!